oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java

```diff
@@ -295,15 +322,38 @@ public MongoDocumentStore(MongoClient connection, MongoDatabase db,
         this.nodeLocks = new StripedNodeDocumentLocks();
         this.nodesCache = builder.buildNodeDocumentCache(this, nodeLocks);
 
+        // if throttling is enabled
+        if (throttleDocumentStore) {
+            this.localDb = connection.getDatabase("local");
+            final MongoIterable<String> collectionNames = localDb.listCollectionNames();
+            String ol = null;
+            for (String e: collectionNames) {
+                if (Objects.equals(e, OPLOG_RS)) {
+                    ol = OPLOG_RS;
+                    break;
+                }
+            }
```

Comment:
Could use Guava instead.
```suggestion
            String ol = Iterables.tryFind(localDb.listCollectionNames(), 
                    s -> Objects.equals(s, OPLOG_RS)).orNull();
```